kuma/wiki/jinja2/wiki/edit.html

```diff
@@ -156,7 +156,7 @@
 
         </header>
 
-        {% if request.user.wiki_revisions().count() == 0 and not revision_form.errors %}
+        {% if not revision_form.errors and not request.user.created_revisions.exists() %}
```

Comment:
This got changed from `user.wiki_revisions()` to `user.created_revisions` ? Is that on purpose?